README.md

```diff
@@ -194,6 +194,8 @@ Please certify your [Developer Certificate of Origin (DCO)](https://developercer
 
 :nerd_face: = nerdctl specific
 
+:window: = Windows enabled
```

Comment:
This emoji doesn't seem included in Windows 10 (21H1)... 😅

Reply:
Maybe we should just use `田` ?

Reply:
Maybe ⊞ (SQUARED PLUS) is better than 田 (CJK UNIFIED IDEOGRAPH-7530)

https://en.wiktionary.org/wiki/%E2%8A%9E

Reply:
You could invert it, and use 🧱 commands that don't run on Windows.

Reply:
ah sorry about that!

Reply:
> 田

This means 'field' and is a very common in both Chinese and Japanese, such as 田鼠 (fieldmouse) or even in brand names 田宫. Best to avoid use of CJK characters.